Mathlib/Topology/Instances/AddCircle.lean

```diff
@@ -125,6 +125,24 @@ abbrev AddCircle [LinearOrderedAddCommGroup 𝕜] [TopologicalSpace 𝕜] [Order
   𝕜 ⧸ zmultiples p
 #align add_circle AddCircle
 
+-- Porting note: the following section replaces a failing `deriving` statement
+section instances
+
+variable [LinearOrderedAddCommGroup 𝕜] [TopologicalSpace 𝕜] [OrderTopology 𝕜] (p : 𝕜)
+
+-- instance : AddCommGroup (AddCircle p) :=
+--   inferInstance -- As (AddCommGroup (𝕜 ⧸ zmultiples p))
+-- instance : TopologicalSpace (AddCircle p) :=
+--   inferInstanceAs (TopologicalSpace (𝕜 ⧸ zmultiples p))
+-- instance : TopologicalAddGroup (AddCircle p) :=
+--   inferInstanceAs (TopologicalAddGroup (𝕜 ⧸ zmultiples p))
+-- instance : Inhabited (AddCircle p) :=
+--   inferInstanceAs (Inhabited (𝕜 ⧸ zmultiples p))
+
+instance : Coe 𝕜 (AddCircle p) := ⟨QuotientAddGroup.mk⟩
+
+end instances
```

Comment:
Are these deliberately commented out? I think this might be the result of a bad merge, and the entire section can be deleted.

Reply:
Sorry, those should have been deleted.